client/src/components/RankingList.jsx

```diff
@@ -0,0 +1,38 @@
+import { useState } from 'react';
+import RankingListQueryButtons from './RankingListQueryButtons';
+import RankingQueryResultList from './RankingQueryResultList';
+import mockRankings from '../mockRankings';
+
+function RankingList() {

```

Comment:
Én ezt a komponents megszűntetném és ami ebben van az mehetne a Rankings.jsx-be közvelenül